src/psyclone/tests/config_test.py

```diff
@@ -571,7 +590,7 @@ def test_invalid_access_mapping(tmpdir):
     'read', 'write', ...) raises an exception.
     '''
     # Test for an invalid key
```

Comment:
Please add a test that the order of the access mappings in the config file doesn't matter.

Reply:
I added a new test, `test_access_mapping_order`, rather than a new section in `test_invalid_access_mapping` as it tests for something different but valid nevertheless. Note, the access mappings testing needs to be tightened as duplicate or mssing key-value pairs do not currently cause failure. I created a new issue #710 to fix this and referenced it in `configuration.py` rather than opening a new can of worms here. I hope you don't mind 😃

Reply:
A new issue is probably the right way to go so I'm happy :-)